ironfish-rust/src/keys/mod.rs

```diff
@@ -194,10 +197,9 @@ impl SaplingKey {
     /// a seed. This isn't strictly necessary for private key, but view keys
     /// will need a direct mapping. The private key could still be generated
     /// using bip-32 and bip-39 if desired.
-    pub fn words_spending_key(&self, language_code: &str) -> Result<String, IronfishError> {
-        let language = Language::from_language_code(language_code)
-            .ok_or(IronfishError::InvalidLanguageEncoding)?;
-        let mnemonic = Mnemonic::from_entropy(&self.spending_key, language).unwrap();
+    pub fn words_spending_key(&self, language: &Language) -> Result<String, IronfishError> {
+        let mnemonic = Mnemonic::from_entropy(&self.spending_key, *language)
```

Comment:
Since `from_entropy` takes the `Language` struct directly, we should just take it in here as well instead of taking the reference. You'll have to remove a few `&` up the chain, but it works otherwise 

```suggestion
    pub fn words_spending_key(&self, language: Language) -> Result<String, IronfishError> {
        let mnemonic = Mnemonic::from_entropy(&self.spending_key, language)
```